perun-core/src/main/java/cz/metacentrum/perun/core/bl/FacilitiesManagerBl.java

```diff
@@ -782,16 +781,64 @@
 	List<Facility> getFacilitiesWhereUserIsAdmin(PerunSession sess, User user) throws InternalErrorException;
 
 	/**
-	 * !!! Not Complete yet, need to implement all perunBeans !!!
+	 * Returns list of facilities connected with a group
 	 *
-	 * Get perunBean and try to find all connected Facilities
+	 * @param sess
+	 * @param group
+	 * @return list of facilities connected with group
+	 * @throws InternalErrorException
+	 */
+	List<Facility> getFacilitiesByPerunBean(PerunSession sess, Group group) throws InternalErrorException;
+
+	/**
+	 * Returns list of facilities with a member
```

Comment:
Connected with a member. Or "by member".

Reply:
I overlooked it, thanks, it should be fixed now.